scripts/utils.ps1

```diff
@@ -62,6 +62,47 @@ function Digest-Hash($path) {
     return Invoke-Expression "md5sum $path"
 }
 
+function Get-Version-Str($file) {
```

Comment:
You should call this `Get-VersionString` to keep to powershells naming convention.

Reply:
Thanks! What a weird convention.